docs/features/wait/http.md

```diff
@@ -21,6 +21,12 @@ Variations on the HTTP wait strategy are supported, including:
 [Waiting for an HTTP endpoint](../../../wait/http_test.go) inside_block:waitForHTTP
 <!--/codeinclude-->
 
+## Match an HTTP method with Port
+
+<!--codeinclude-->
+[Waiting for an HTTP endpoint](../../../wait/http_test.go) inside_block:waitForHTTPWithPort
```

Comment:
For some reason, this locator it causing issues when rendering the mkdocs site: https://deploy-preview-1086--testcontainers-go.netlify.app/features/wait/http/

Could you check why? I'd also include a paragraph explaining what you described in the PR description:

> Instead of using port 80 as a default, HTTP wait strategy contains no default. If the user does not set an explicit port, the first exposed port is automatically used for liveness checking.

I think a good place would be right before the `variations` section.

You can simply test it locally using the instructions here: https://deploy-preview-1086--testcontainers-go.netlify.app/contributing_docs/#previewing-rendered-content

Reply:
Thanks for these pointers - sorry about it will take a look next week

Reply:
Hey @anuraaga please let me know if I can be of any help with the docs here 🙏  I'd like to include this fix into the project :)

Reply:
@anuraaga I'm going ahead and fixing this by myself. If you don't mind, I'm going to push my commits right after yours.